backend/Dockerfile

```diff
@@ -1,15 +1,14 @@
 # syntax=docker/dockerfile:1
-FROM ruby:3.1.2
-RUN apt-get update -y && apt-get upgrade -y && apt-get install -y postgresql-client
-# for alpine; when dependency resolution is fixed.
-# FROM ruby:3.1.2-alpine
-# RUN apk update && apk upgrade && apk add postgresql-client
+FROM ruby:3.1.3-alpine
+RUN apk update && apk upgrade && apk add build-base libpq-dev postgresql-client gcompat tzdata
 WORKDIR /usr/src/app
 COPY Gemfile Gemfile.lock ./
+# add x86_64 for support on ARM platform - nokogiri expects x86 for distribution
+RUN bundle lock --add-platform x86_64-linux
```

Comment:
If your using ARM shouldn't this be `aarch64-linux`? It's a [supported platform for nokogiri](https://nokogiri.org/tutorials/installing_nokogiri.html#supported-platforms).  

That said `x86_64-linux` is [already available in the current Gemfile.lock](https://github.com/codeforboston/urban-league-heat-pump-accelerator/blob/main/backend/Gemfile.lock#LL241) so you shouldn't have to run this every time.